kani-compiler/src/kani_middle/mod.rs

```diff
@@ -50,6 +54,23 @@ pub fn check_crate_items(tcx: TyCtxt, ignore_asm: bool) {
     tcx.sess.abort_if_errors();
 }
 
+/// Check that all given items are supported and there's no misconfiguration.
+/// This method will exhaustively print any error / warning and it will abort at the end if any
+/// error was found.
+pub fn check_reachable_items(tcx: TyCtxt, queries: &QueryDb, items: &[MonoItem]) {
```

Comment:
The function name and description are a bit too general. Do you foresee expanding the checks that this function does in the future? If not, I would suggest renaming it to something more specific, e.g. `check_reachable_unstable_items`.

Reply:
Yes, I was thinking that this will likely be extended in the future. But I'm OK renaming it for now if you prefer.

Reply:
Either way